frontend/view-edit-client/interactions/single-interaction-slat.component.tsx

```diff
@@ -335,14 +331,95 @@ export default React.forwardRef<any, SingleClientInteractionProps>(
         </div>
       </div>
     );
+
+    function updateInteractionKind(evt: React.ChangeEvent<HTMLInputElement>) {
+      setInteractionKind(evt.target.value as InteractionKind);
+    }
+
+    function cuServiceSelect() {
+      return (
+        <select
+          ref={ref}
+          value={
+            selectedService
+              ? selectedService.type + selectedService.service.id
+              : ""
+          }
+          onChange={(evt) => {
+            const serviceId = Number(evt.target.value.slice("CU".length));
+            setSelectedService({
+              type: "CU",
+              service: services.find((s) => s.id === serviceId),
+            });
+          }}
+          aria-labelledby={`provided-service-${props.interactionIndex}`}
+          className="services-select"
+          name={`provided-service-${props.interactionIndex}`}
+          required
+        >
+          <option value="" disabled hidden>
+            Choose here
+          </option>
+          {Object.keys(groupedServices).map((programName) => (
+            <optgroup label={programName} key={programName}>
+              {groupedServices[programName].map((service) => (
+                <option key={service.id} value={"CU" + service.id}>
+                  {service.serviceName}
+                </option>
+              ))}
+            </optgroup>
+          ))}
+        </select>
+      );
+    }
+
+    function referralSelect() {
+      return (
+        <select
+          ref={ref}
+          value={
+            selectedService
+              ? selectedService.type + selectedService.service.id
+              : ""
+          }
+          onChange={(evt) => {
+            const serviceId = Number(evt.target.value.slice("Partner".length));
+            setSelectedService({
+              type: "Partner",
+              service: partnerServices.find((ps) => ps.id === serviceId),
+            });
+          }}
+          aria-labelledby={`provided-service-${props.interactionIndex}`}
+          className="services-select"
+          name={`provided-service-${props.interactionIndex}`}
+          required
+        >
+          <option value="" disabled hidden>
+            Choose here
+          </option>
+          {props.partnersResponse.map((partner) => (
+            <optgroup label={partner.name + " (Referral)"} key={partner.id}>
+              {partner.services.map((partnerService) => (
+                <option
+                  key={partnerService.id}
+                  value={"Partner" + partnerService.id}
+                >
+                  {partnerService.name}
+                </option>
+              ))}
+            </optgroup>
+          ))}
+        </select>
+      );
+    }
   }
 );
 
 function getName(service: AnyService) {
   if (service.type === "CU") {
     return service.service.serviceName;
   } else {
-    return service.service.name + " (Referral)";
```

Comment:
We no longer need to put (Referral) into the title, since there is the question now just below the title that shows it clearly